x-pack/plugins/security_solution/cypress/e2e/detection_rules/prebuilt_rules_notifications.cy.ts

```diff
@@ -55,64 +58,123 @@ describe('Detection rules, Prebuilt Rules Installation and Update workflow', ()
     });
   });
 
-  describe('Rule installation notification when at least one rule already installed', () => {
+  describe('Notifications', () => {
     beforeEach(() => {
-      installAllPrebuiltRulesRequest();
-      /* Create new rule assets with a different rule_id as the one that was */
-      /* installed before in order to trigger the installation notification */
-      const RULE_2 = createRuleAssetSavedObject({
-        name: 'Test rule 2',
-        rule_id: 'rule_2',
-      });
-      const RULE_3 = createRuleAssetSavedObject({
-        name: 'Test rule 3',
-        rule_id: 'rule_3',
+      createAndInstallMockedPrebuiltRules({ rules: [RULE_1], installToKibana: false });
+    });
+
+    describe('Rules installation notification when no rules have been installed', () => {
+      beforeEach(() => {
```

Comment:
Similar to previous comments, here we have three levels of `beforeEach` blocks making it very hard to understand the initial conditions of the test cases. My recommendation would be to restructure these tests for clarity.